.gitignore

```diff
@@ -3,6 +3,7 @@ node_modules
 /dist
 
 # local env files
+.env
```

Comment:
I believe all the contents of this .gitignore are what were generated when Vue CLI created the app. Strange it didn't include .env